source/data/data_discovery.rst

```diff
@@ -25,25 +25,25 @@ In addition to supporting data discovery, metadata also organizes and provides c
 Create Attribute, Value, Unit triples
 -------------------------------------
 
-iRODS provides the user with the possibility to create Attribute Value Unit triplets and store them with the data. The triplets are stored in the iCAT catalogue, which can be queried to identify and retrieve the correct objects.
-We can annotate files with so-called AVUs triples. These triples are added to a database and are searchable, so that we will be able to ask the iRODS system give me all data (files and collections) based on the matching query criteria.
-First we will explore how to create these AVUs triples for which we can search later.
+iRODS provides the user with the possibility to create Attribute-Value-Unit (AVU) triples and associate them with the data. The triplets are stored in the iCAT catalogue (in the database), which can be queried to identify and retrieve the correct objects.
```

Comment:
I guess this part could include collections and any data objects and some typos could be corrected. My suggestion is:

> iRODS provides users with the possibility to create Attribute-Value-Unit (AVU) triples and associate them with collections and data objects. The triplets are stored in the iCAT catalogue (in the database), which can be queried to identify and retrieve the correct objects when needed.

An correctly we are not mentioning about attaching metadata to other entities/objects exists in irods such as users, resources etc.